packages/repository/src/repositories/relation.repository.ts

```diff
@@ -55,41 +56,55 @@ export interface HasManyRepository<Target extends Entity> {
   ): Promise<number>;
 }
 
+export interface BelongsToRepository<Target extends Entity> {
```

Comment:
> AFAICT, this interface is not used at all, because BelongsToFactory is returning Promise<Target>. Can we remove it?

FWIW, this comment is no longer relevant.